src/cryptoadvance/specter/controller.py

```diff
@@ -1960,3 +1960,111 @@ def notify_upgrade():
             "info",
         )
     return app.specter.version.current
+
+
+
+############### API endpoints ##################
```

Comment:
Not a must but would be really nice to have this in a separate file, ideally as a separate [Blueprint](https://realpython.com/flask-blueprint/) like we have the HWIBridge stuff (see `hwi_server.py`).

Reply:
For a next update I'm planning to:
- create a separate `api.py` file with a separate blueprint. Agree it's the best alternative.
- Explore using Flask-restful as suggested by @k9ert